src/buttons.css

```diff
@@ -147,18 +144,15 @@
  *
  * @memberof Buttons
  * @example
- * <div>
- *   <div class="mr12 flex-parent-inline">
- *     <button class="btn btn--pill btn--pill-hl">Left</button>
- *     <button class="btn btn--pill btn--pill-hc">Center</button>
- *     <button class="btn btn--pill btn--pill-hr">Right</button>
- *   </div>
+ * <button class="btn btn--pill btn--pill-hl">Left</button>
+ * <button class="btn btn--pill btn--pill-hc">Center</button>
+ * <button class="btn btn--pill btn--pill-hr">Right</button>
```

Comment:
If I try adding `btn--stroke` to this class list we get this space between the buttons:

![screen shot 2017-01-04 at 5 59 05 pm](https://cloud.githubusercontent.com/assets/628431/21665056/a0466bd2-d2a7-11e6-8628-16385747b3d6.png)